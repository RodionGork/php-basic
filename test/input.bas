input "x=", x
print "x*x=";x*x
input "x=", x, "y=", y
print "x+y=";x+y
input "", "line: ", line
print len(line)
input "word1=", word1, "word2=", word2
print word2, word1