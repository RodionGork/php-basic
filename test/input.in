71
85 96
zaec pakemon
hare wolf