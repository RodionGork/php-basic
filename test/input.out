x=x*x=5041
x=y=x+y=181
line: 12
word1=word2=wolf hare
